packages/react/src/lib/positioner/types.ts

```diff
@@ -6,12 +6,21 @@ export type Alignment = 'top' | 'bottom' | 'start' | 'end' | 'center'
 
 export type PopperChildrenFn = (props: PopperChildrenProps) => React.ReactNode
 
-export interface PositioningProps {
+export interface SimplifiedPositioningProps {
```

Comment:
NIT: another recommendation would be `BasicPositioningProps` but I'm ok with any of them